src/services/UserService.js

```diff
@@ -0,0 +1,13 @@
+import config from '../config/config.json'
+import axios from 'axios'
+
+const baseUrl = config.baseUrl;
+
+class UserService {
+
+    register = user => {
+        return axios.post(`${baseUrl}todoapp/register`, user);
```

Comment:
We can create some basic centralized services - for authentication and HTTP methods. I will give you more details on Slack.